puppet-checks/src/test/groovy/com/iadams/sonarqube/puppet/checks/IndentationCheckSpec.groovy

```diff
@@ -53,35 +50,25 @@ class IndentationCheckSpec extends Specification {
-      .next().atLine(64).withMessage("Make this line start at column 5.")
-      .next().atLine(70).withMessage("Make this line start at column 3.")
-      .next().atLine(71).withMessage("Make this line start at column 3.")
-      .next().atLine(77).withMessage("Make this line start at column 5.")
-      .next().atLine(78).withMessage("Make this line start at column 5.")
+      .next().atLine(58).withMessage("The following 1 lines should start on column 3.")
+      .next().atLine(63).withMessage("The following 1 lines should start on column 5.")
+      .next().atLine(70).withMessage("The following 1 lines should start on column 3.")
+      .next().atLine(77).withMessage("The following 1 lines should start on column 5.")
       .next().atLine(79).withMessage("Make this line start at column 3.")
-      .next().atLine(87).withMessage("Make this line start at column 3.")
-      .next().atLine(88).withMessage("Make this line start at column 3.")
-      .next().atLine(93).withMessage("Make this line start at column 3.")
-      .next().atLine(94).withMessage("Make this line start at column 3.")
+      .next().atLine(87).withMessage("The following 1 lines should start on column 3.")
+      .next().atLine(93).withMessage("The following 1 lines should start on column 3.")
       .next().atLine(100).withMessage("Make this line start at column 5.")
       .next().atLine(103).withMessage("Make this line start at column 3.")
-      .next().atLine(108).withMessage("Make this line start at column 3.")
-      .next().atLine(109).withMessage("Make this line start at column 3.")
+      .next().atLine(108).withMessage("The following 1 lines should start on column 3.")
       .next().atLine(114).withMessage("Make this line start at column 3.")
       .next().atLine(117).withMessage("Make this line start at column 5.")
-      .next().atLine(125).withMessage("Make this line start at column 5.")
-      .next().atLine(126).withMessage("Make this line start at column 5.")
-      .next().atLine(132).withMessage("Make this line start at column 7.")
-      .next().atLine(133).withMessage("Make this line start at column 7.")
-      .next().atLine(136).withMessage("Make this line start at column 5.")
-      .next().atLine(137).withMessage("Make this line start at column 5.")
-      .next().atLine(141).withMessage("Make this line start at column 1.")
-      .next().atLine(142).withMessage("Make this line start at column 1.")
-      .next().atLine(143).withMessage("Make this line start at column 1.")
+      .next().atLine(125).withMessage("The following 1 lines should start on column 5.")
+      .next().atLine(132).withMessage("The following 1 lines should start on column 7.")
+      .next().atLine(136).withMessage("The following 1 lines should start on column 5.")
+      .next().atLine(141).withMessage("The following 2 lines should start on column 1.")
+      .next().atLine(293).withMessage("The following 3 lines should start on column 3.")
```

Comment:
It should be the following 4 lines (instead of 3, 293, 294, 295 and 296), shouldn't it?